### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [ ] Release 1.2.0
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/nf-core/gawk/main.nf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        gawk: \$(awk -Wversion | sed '1!d; s/.*Awk //; s/,.*//')
+    END_VERSIONS
+    """
+
+    stub:
+    prefix = task.ext.prefix ?: "${meta.id}"
+    suffix = task.ext.suffix ?: "${input.getExtension}"
```
Comment (Lucpen) [inline modules/nf-core/gawk/main.nf:44]:
```suggestion
    suffix = task.ext.suffix ?: "${input.getExtension()}"
```

File: modules.json [hunk 2]
```diff
@@ -81,6 +86,11 @@
                         "git_sha": "d742e3143f2ccb8853c29b35cfcf50b5e5026980",
                         "installed_by": ["modules"]
                     },
+                    "gawk": {
+                        "branch": "master",
+                        "git_sha": "da4d05d04e65227d4307e87940842f1a14de62c7",
+                        "installed_by": ["modules"]
```
Comment (Lucpen) [inline modules.json:92]:
```suggestion
                        "git_sha": "e8dd194205c8657440afc34695e3bb2d32840ce0",
                        "installed_by": ["modules"]
```

**Comment by rannick** [comment]
I would make this a major release as the parameters changed and usage is not backwards compatible

File: nextflow.config [hunk 3]
```diff
@@ -32,25 +32,25 @@ params {
...
     drop_group_samples_ae      = 'outrider'
     drop_group_samples_as      = 'fraser'
     drop_padjcutoff_ae         = 0.05
     drop_padjcutoff_as         = 0.1
     drop_zscorecutoff          = 0
-    switch_vep                 = true
+    skip_vep                   = false
 
     // Variant annotation
     vep_cache_version          = 110
```
[the remark this answers is not in the document]
Comment (Lucpen) [reply nextflow.config:56]: It's only 107 on the test data but should be 110 on the real data set

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,17 +3,66 @@
...
+## 1.2.1 - [XXXX-XX-XX]
+
+### `Added`
+
+### `Fixed`
+
+### `Parameters`
+
+## 1.2.0 - Santa [2024-04-19]
+
```
Comment (Lucpen) [inline CHANGELOG.md:26]:
```suggestion

## 2.0.1 - [XXXX-XX-XX]

### `Added`

### `Fixed`

### `Parameters`

## 2.0.0 - Santa [2024-04-19]

```

File: assets/multiqc_config.yml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/genomic-medicine-sweden/tomte/releases/tag/1.1.0" target="_blank">genomic-medicine-sweden/tomte</a>
+  This report has been generated by the <a href="https://github.com/genomic-medicine-sweden/tomte/releases/tag/1.2.0" target="_blank">genomic-medicine-sweden/tomte</a>
```
Comment (Lucpen) [inline assets/multiqc_config.yml:2]:
```suggestion
  This report has been generated by the <a href="https://github.com/genomic-medicine-sweden/tomte/releases/tag/2.0.0" target="_blank">genomic-medicine-sweden/tomte</a>
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,18 +3,67 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Parameters
+
+| Old parameter | New parameter |
+| ------------- | ------------- |
+
+> [!NOTE]
```
Comment (Lucpen) [inline CHANGELOG.md:11]:
```suggestion
> [!NOTE]
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,18 +3,67 @@
...
+> Parameter has been updated if both old and new parameter information is present.
+> Parameter has been added if just the new parameter information is present.
+> Parameter has been removed if new parameter information isn't present.
+
```
Comment (Lucpen) [inline CHANGELOG.md:15]:
```suggestion

:::note
Parameter has been updated if both old and new parameter information is present.
Parameter has been added if just the new parameter information is present.
Parameter has been removed if new parameter information isn't present.
:::

```

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,18 +3,67 @@
...
+| `--switch_subsample_region` | `--skip_subsample_region` |
+| `--switch_downsample`       | `--skip_downsample`       |
+| `--switch_build_tracks`     | `--skip_build_tracks`     |
+| `--switch_stringtie`        | `--skip_stringtie`        |
+| `--switch_vep`              | `--skip_vep`              |
+| `--switch_drop_ae`          | `--skip_drop_ae`          |
+| `--switch_drop_as`          | `--skip_drop_as`          |
+
+> [!NOTE] 
+> Parameter has been updated if both old and new parameter information is present. Parameter has been added if just the new parameter information is present. Parameter has been removed if new parameter information isn't present.
```
Comment (Lucpen) [inline CHANGELOG.md:66]:
```suggestion
> [!NOTE]
> Parameter has been updated if both old and new parameter information is present. Parameter has been added if just the new parameter information is present. Parameter has been removed if new parameter information isn't present.
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -3,18 +3,67 @@
...
+| `--switch_subsample_region` | `--skip_subsample_region` |
+| `--switch_downsample`       | `--skip_downsample`       |
+| `--switch_build_tracks`     | `--skip_build_tracks`     |
+| `--switch_stringtie`        | `--skip_stringtie`        |
+| `--switch_vep`              | `--skip_vep`              |
+| `--switch_drop_ae`          | `--skip_drop_ae`          |
+| `--switch_drop_as`          | `--skip_drop_as`          |
+
+> [!NOTE]
+> Parameter has been updated if both old and new parameter information is present. Parameter has been added if just the new parameter information is present. Parameter has been removed if new parameter information isn't present.
```
Comment (Lucpen) [inline CHANGELOG.md:66]:
```suggestion
:::note Parameter has been updated if both old and new parameter information is present. Parameter has been added if just the new parameter information is present. Parameter has been removed if new parameter information isn't present. :::
```

**Review by rannick — APPROVED** [review]
LGTM, parameters are much more readable

**Pull request merged**